src/mon/MgrStatMonitor.h

```diff
@@ -56,8 +56,11 @@
 
   void calc_pool_availability();
   bool enable_availability_tracking = g_conf().get_val<bool>("enable_availability_tracking"); ///< tracking availability score feature 
-  
+  double pool_availability_update_interval = g_conf().get_val<double>("pool_availability_update_interval");
```

Comment:
yeah here it's `get_val<double>` so `handle_conf_change` should also be `double`